diff --git a/llava/llava/model/__init__.py b/llava/llava/model/__init__.py
index 12a997f..7baac2f 100644
--- a/llava/llava/model/__init__.py
+++ b/llava/llava/model/__init__.py
@@ -1,4 +1,4 @@
 from .language_model.llava_llama import LlavaLlamaForCausalLM, LlavaConfig
 from .language_model.llava_mistral import LlavaMistralForCausalLM, LlavaConfig
 from .language_model.llava_mpt import LlavaMPTForCausalLM, LlavaMPTConfig
-from .language_model.llava_stablelm import LlavaStableLMEpochForCausalLM, LlavaStableLMEpochConfig
+from .language_model.llava_stablelm import LlavaStableLMEpochForCausalLM, LlavaConfig
diff --git a/llava/llava/model/__pycache__/__init__.cpython-310.pyc b/llava/llava/model/__pycache__/__init__.cpython-310.pyc
index b31b049..f1dba84 100644
Binary files a/llava/llava/model/__pycache__/__init__.cpython-310.pyc and b/llava/llava/model/__pycache__/__init__.cpython-310.pyc differ
diff --git a/llava/llava/model/language_model/__pycache__/llava_stablelm.cpython-310.pyc b/llava/llava/model/language_model/__pycache__/llava_stablelm.cpython-310.pyc
index 446abdd..97aa09c 100644
Binary files a/llava/llava/model/language_model/__pycache__/llava_stablelm.cpython-310.pyc and b/llava/llava/model/language_model/__pycache__/llava_stablelm.cpython-310.pyc differ
diff --git a/llava/llava/model/language_model/llava_stablelm.py b/llava/llava/model/language_model/llava_stablelm.py
index 9b4533b..b4dccd8 100644
--- a/llava/llava/model/language_model/llava_stablelm.py
+++ b/llava/llava/model/language_model/llava_stablelm.py
@@ -25,22 +25,22 @@ from transformers.modeling_outputs import CausalLMOutputWithPast
 from .stablelm.modeling_stablelm_epoch import StableLMEpochConfig, StableLMEpochForCausalLM, StableLMEpochModel
 from ..llava_arch import LlavaMetaModel, LlavaMetaForCausalLM
 
-class LlavaStableLMEpochConfig(StableLMEpochConfig):
+class LlavaConfig(StableLMEpochConfig):
     model_type = "llava_stablelm_epoch"
 
 
 class LlavaStableLMEpochModel(LlavaMetaModel, StableLMEpochModel):
-    config_class = LlavaStableLMEpochConfig
+    config_class = LlavaConfig
 
-    def __init__(self, config: LlavaStableLMEpochConfig):
+    def __init__(self, config: LlavaConfig):
         super(LlavaStableLMEpochModel, self).__init__(config)
 
 class LlavaStableLMEpochForCausalLM(StableLMEpochForCausalLM, LlavaMetaForCausalLM):
-    config_class = LlavaStableLMEpochConfig
+    config_class = LlavaConfig
 
     def __init__(self, config):
         super(StableLMEpochForCausalLM, self).__init__(config)
-        self.model = StableLMEpochModel(config)
+        self.model = LlavaStableLMEpochModel(config)
 
         self.lm_head = nn.Linear(config.hidden_size, config.vocab_size, bias=False)
 
@@ -133,5 +133,5 @@ class LlavaStableLMEpochForCausalLM(StableLMEpochForCausalLM, LlavaMetaForCausal
         )
         return model_inputs
 
-AutoConfig.register("llava_stablelm_epoch", LlavaStableLMEpochConfig)
-AutoModelForCausalLM.register(LlavaStableLMEpochConfig, LlavaStableLMEpochForCausalLM)
+AutoConfig.register("llava_stablelm_epoch", LlavaConfig)
+AutoModelForCausalLM.register(LlavaConfig, LlavaStableLMEpochForCausalLM)
diff --git a/llava/llava/model/language_model/mpt/__pycache__/configuration_mpt.cpython-310.pyc b/llava/llava/model/language_model/mpt/__pycache__/configuration_mpt.cpython-310.pyc
index 52819e6..a51c7c3 100644
Binary files a/llava/llava/model/language_model/mpt/__pycache__/configuration_mpt.cpython-310.pyc and b/llava/llava/model/language_model/mpt/__pycache__/configuration_mpt.cpython-310.pyc differ
diff --git a/llava/llava/train/__pycache__/train.cpython-310.pyc b/llava/llava/train/__pycache__/train.cpython-310.pyc
index fe61b36..c2fa001 100644
Binary files a/llava/llava/train/__pycache__/train.cpython-310.pyc and b/llava/llava/train/__pycache__/train.cpython-310.pyc differ
diff --git a/llava/scripts/pretrain.sh b/llava/scripts/pretrain.sh
index 7897493..24e495b 100644
--- a/llava/scripts/pretrain.sh
+++ b/llava/scripts/pretrain.sh
@@ -4,7 +4,7 @@
 
 # MODEL_VERSION=vicuna-v1-3-7b
 # MODEL_VERSION=llama-2-7b-chat
-MODEL_VERSION=stabilityai/stablelm-3b-4e1t
+MODEL_VERSION=mistralai/Mistral-7B-v0.1
 
 ########### DO NOT CHANGE ###########
 ########### USE THIS FOR BOTH ###########
diff --git a/llava/scripts/pretrain_stablelm.sh b/llava/scripts/pretrain_stablelm.sh
index 7897493..e873b06 100644
--- a/llava/scripts/pretrain_stablelm.sh
+++ b/llava/scripts/pretrain_stablelm.sh
@@ -20,7 +20,7 @@ deepspeed llava/train/train_mem.py \
     --deepspeed ./scripts/zero2.json \
     --model_name_or_path $MODEL_VERSION \
     --version $PROMPT_VERSION \
-    --data_path pretrain_./chat.json \
+    --data_path ./chat.json \
     --image_folder ./train_images \
     --vision_tower openai/clip-vit-large-patch14-336 \
     --tune_mm_mlp_adapter True \
